test/Microsoft.Azure.Devices.Edge.Test.Common/Windows/EdgeDaemon.cs

```diff
@@ -8,13 +8,7 @@ namespace Microsoft.Azure.Devices.Edge.Test.Common
     using Microsoft.Azure.Devices.Edge.Util;
     using Serilog;
 
-    public enum EdgeDaemonStatus
-    {
-        Running = ServiceControllerStatus.Running,
-        Stopped = ServiceControllerStatus.Stopped
-    }
-
-    public class EdgeDaemon
+    public class EdgeDaemon : IEdgeDaemon
```

Comment:
Nit: right now there are 2 EdgeDaemon classes under 2 different folders, rename this to WindowsEdgeDaemon? This will make looking up class using Resharper a bit easier.

Reply:
Yeah, that's intentional...I preferred using the namespace to differentiate, rather than loading up the class name. Generally when you do lookup with resharper and other tools it will show the namespace as well (or at least the folder), so I don't think that's a problem.